keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ImageResolver.kt

```diff
@@ -47,12 +48,12 @@ class ImageResolver(
         }
         val account = dynamicConfigService.getConfig("images.default-account", "test")
         val namedImage = imageService
-          .getLatestNamedImage(artifactName, account) ?: throw NoImageFound(artifactName)
+          .getLatestNamedImage(artifactName, account, region) ?: throw NoImageFound(artifactName)
```

Comment:
supplying the cluster resource's region here means we'll get a filtered list of AMIs back from River